wiki/Ranking_Criteria/en.md

```diff
@@ -143,11 +143,10 @@ Changes to the rules and guidelines in this document are proposed and discussed
 
 ### Allowances
 
-This category contains explicit allowance statements of concepts and rules that are not commonly straightforward even after reading this whole section of the ranking criteria.
+This category contains explicit allowance statements of concepts and rules that are not commonly straightforward even after reading this whole section of the ranking criteria. In cases where there are multiple options available for a song's metadata, the above standardisation rules for the Ranked and Loved sections take priority.
 
 - **For songs where the composer(s) and singer(s) are different people, the singer(s) may be listed after the composer(s) or circle/group name following a `feat.` indicator.**
-- **If an artist has provided an official translation for their name, this may be used in the romanised artist field.** Official romanisation may be used for the spelling of an artist's name, but the name order must follow the related rule.
-- **If a Unicode Song title has either an official translation or romanisation provided by the artist, either or may be used in the romanised title field.**
+- **If a Unicode song title or artist has either an official translation or romanisation provided by the artist, either or may be used in the respective romanised field.**
```

Comment:
either or? the artist is specifying two? am i english?

Reply:
```suggestion
- **If a Unicode song title or artist has either an official translation or romanisation provided by the artist, it may be used in the respective romanised field.**
```
?

Reply:
Song titles often do have both, such as https://www.youtube.com/watch?v=XZnmfDByGWU or many others (first example that came to mind). It's less common for artists.

Reply:
the flow chart changes nothing. it's just splitting the sentence into boxes. the point is that it is ambiguous to someone reading it

"oh either may be used? i think i'll choose the one that is different from 100 ranked maps just to be difficult!"

is this not something we want to avoid? the sentence should say which is preferred to give the reader direction. if the preference is to follow suite then that should be mentioned.

Reply:
`If the same song exists in the Ranked or Loved sections already, the metadata should be followed unless it breaks other rules in the ranking criteria or the official sources state something completely different.` and 
`**Artist names should be consistent between different songs from the same person or group in the Ranked or Loved sections.** This does not apply if the person or group intentionally uses a different alias for different song or album releases. ` 

are in standardization guidelines already since consistency across sets relates to any meta rules or cases where there's options and not just romanisation.

Reply:
So this is the case where this is the first beatmap by the artist that is ranked? In that case, shouldn't the official romanisation be used?

Reply:
If that's the case, that expands beyond just how romanisation is handled and is out of scope of this rule. The topic of pushing what options for metadata in general is preferred would need a seperate community discussion first and creating it as its own separate guideline, to make sure it's something agreed upon and easily understood. This would be instead of putting it in solely for artist name romanisation.

Reply:
whether this should explicitly be a choice or preferred one way, i'll just put this here to get the sentence into a readable state in case this does go through.

```suggestion
- **If a Unicode song title or artist has an official translation or romanisation provided by the artist, either may be used in the respective romanised field.**
```

Reply:
I still don't know what "either" is referencing here. There's no two items in the adjacent text. Is it talking about the "feat" term?

Reply:
Either the translation or the romanisation, as mentioned some songs and artists have and use both translation and romanisation interchangeably.

Reply:
"either [of them]" is referring to the "translation or the romanisation" in the same sentence, and to me that reads fine, but here's an alternative suggestion:

```suggestion
- **Official translations or romanisations of the Unicode song title or artist name may be used in their respective romanised fields.**
```

Reply:
"either" in english can only be used if there are two optiond at hand. please re-read my issue.

If this is talking about a case where the artist/title has two options, this sentence needs to mention what the two options are.

Reply:
hmm. maybe i've misunderstood what this clause was supposed to convey. @NoffyNoffeh is this clause supposed to only cover the case where both of these options are available?

```suggestion
- **If a Unicode song title or artist has both an official translation or romanisation provided by the artist, either of them may be used in the respective romanised field.**
```

Reply:
@Walavouchey did you mean "official translation *and*"? because at that point the sentence validates for me.

Reply:
Saying and doesn't make sense, that implies both title and artist have to be translated/romanized to be able to use one.

Plenty of artists have translated/romanized names but not song titles.


I don't understand your issue with the original sentence, EITHER translation OR romanisation when both exist. Both options are in the same sentence as when `either` is ever mentioned.

Reply:
1) this only implies that there is no preference between any official translation or romanisation if both are available. it doesn't say anything about any other case.

2) if the issue is with the usage of "either", then this clause simply needs to be reworded to be a bit longer to cover both cases

Reply:
The clause covers using official options when they available.

Its main function is telling you that you can use official options, and mentions if there's both romanisation/translation, use either romanisation or translation. Choosing between the two is not the main point.


I don't understand why a completely different clause about `feat.` is being mentioned when it's totally unrelated and is its own point.

Reply:
we really are getting lost in the wording. i hope this satisfies all sides, and if not, i'll leave this for other reviewers to opinionate over.

```suggestion
- **If a Unicode song title or artist has an official translation or romanisation provided by the artist, it may be used in the respective romanised field. If both are available, either may be used.**
```

Reply:
I'm only getting lost because I know others will in the same way. Not trying to be difficult.

So without reading the previous discussion: This is implying that either the "translation" or "romanisation" may be used?

Reply:
Yes

Reply:
I've reworded the suggestion slightly to be more explicit.